canonical-web-and-design/charmhub.io
# Add bundle overview page

## Done
- _Add bundle overview page._
-  _Show only the `Overview` tab for bundle details page._

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to http://localhost:8045/kubeflow and see the page looks fine: has topology, and you can only see the overview tab
- Go to http://localhost:8045/postgresql-k8s and see everything is the same as https://charmhub.io/postgresql-k8s

## Issue / Card
Fixes #749 

## Screenshots
[if relevant, include a screenshot]
